cache/in-memory/src/model/guild.rs

```diff
@@ -70,7 +70,7 @@ pub struct CachedGuild {
     pub owner_id: UserId,
     /// Total permissions for the current user in the guild, excluding overwrites.
     pub permissions: Option<Permissions>,
-    /// For Community guilds, the preferred locale.
+    /// Preferred locale for Community guilds.
```

Comment:
I made this change to stick with a format `For (X feature that is non-standard), (docs).`

Reply:
I didn't realize we had a format for that

Reply:
It doesn't seem grammatically correct is all

Reply:
We don't have a format for it. But when I document model, I was considering making it a format. Also, it seems grammatically correct to me.